app/main.py

```diff
@@ -2,24 +2,72 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self,
+                     owner: "SlideLimitationValidator",
+                     name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self,
+                instance: "SlideLimitationValidator",
+                owner: "SlideLimitationValidator") -> int:
```

Comment:
Are you sure about your annotations?)
Also try to import annotations from __future__